apps/tangle-dapp/data/ValidatorTables/useValidators.ts

```diff
@@ -103,11 +116,14 @@ export const useValidators = (
     });
   }, [
     addresses,
```

Comment:
Check out this warning from GitHub Actions:

![CleanShot 2024-04-04 at 11 31 34@2x](https://github.com/webb-tools/webb-dapp/assets/69841784/56078083-29ac-4344-a9e9-fe50ed913d22)